compose/ui/ui/src/macosMain/kotlin/androidx/compose/ui/window/ComposeWindow.macos.kt

```diff
@@ -93,6 +95,14 @@ private class ComposeWindow(
         object : PlatformContext by PlatformContext.Empty {
             override val windowInfo get() = _windowInfo
             override val textInputService get() = macosTextInputService
+            override fun setPointerIcon(pointerIcon: PointerIcon) {
+                when (pointerIcon) {
```

Comment:
We should set it to `NSCursor.arrowCursor.set()` in `else` case

Reply:
Fixed, thanks!